### DISCUSSION TITLE
Linux Workshop

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [EddieHubCommunity/support#1732] [Show and tell] [answered]
We're planning on a Linux Workshop hosted by Devs'Street. Some of the details are as follows:

**Time:** 18:00 IST | 12:30 GMT
**Date:** 12-06-2021 (Sunday)
**Duration**: ~2 hours

**Target audience**:
- Those who don't know Linux
- Those who want to get started with Linux
- Those who want to start making minor projects for the OS

**Speakers**:
1. Karuna - @starlightknown 
2. Kaiwalya - @kaiwalyakoparkar 
3. Faisal - @felixfaisal 

Here are the [(canva) slides](https://www.canva.com/design/DAEfx7hjuok/8JMge_Ef_eMV7uKbsPkyTw/view?utm_content=DAEfx7hjuok&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton) and here's the [thread to discuss them](https://github.com/EddieHubCommunity/support/discussions/1732#discussioncomment-838184)

We'll add details in the comments below - We'd love to hear your thoughts, suggestions and feedback!

**Comment by felixfaisal** [comment]
Hey everyone, 
I will be doing a hands-on session on shell scripting, It would be great to hear about scripts you have created to help in automation, productivity or just for fun :D

**Comment by kaiwalyakoparkar** [comment]
Hey everyone,
I will be doing a hands-on session on some terminal commands. It would love to hear about some of your favourite commands that could be included :D

**Comment by Akshu-on-github** [comment, marked as answer]
Check out our [slides](https://www.canva.com/design/DAEfx7hjuok/8JMge_Ef_eMV7uKbsPkyTw/view?utm_content=DAEfx7hjuok&utm_campaign=designshare&utm_medium=link&utm_source=sharebutton) - We've divided the workshop into three segments:
- Introducing Linux (Slides 4 to 10 handled by @starlightknown)
- Linux Commands (Slides 11 to 18 handled by @kaiwalyakoparkar)
- Shell Scripting (Slides 18 to 25 handled by @felixfaisal)

A penny for your thoughts?
- Did we miss anything?
- How's the content? Is it clear, legible?
- What do you think about the design?
- Any likes/dislikes?

**Reply by Akshu-on-github** [reply]
Thank you all for making this event a huge success 😄
I'll close this discussion for now
